tests/std/tests/P0083R3_splicing_maps_and_sets/test.cpp

```diff
@@ -634,6 +658,143 @@ void test_unordered_set() {
     test_merge<Set, std::unordered_multiset>();
 }
 
+void test_gh1309() {
+    // Guard against regression of GH-1309, in which node handles were incorrectly destroying the user value with a node
+    // allocator rather than a value_type allocator as the Standard requires.
+
+    {
+        using A  = tracked_allocator<std::pair<int const, char>>;
+        using M  = std::map<int, char, std::less<>, A>;
+        using NH = decltype(std::declval<M&>().extract(42));

```

Comment:
This is correct, but I believe it would be shorter/simpler to use the `node_type` typedef:
```suggestion
        using NH = M::node_type;
```
Occurs below.